test/contrib/loraphy2wan.uts

```diff
@@ -0,0 +1,23 @@
+% Regression tests for Scapy
```

Comment:
It will be awesome if you could add basic building & dissecting unit tests for the packets that you implemented, such as https://github.com/secdev/scapy/blob/master/test/contrib/gtp.uts#L129

Reply:
Few more dissect tests + building tests a suggested have been added to the file